src/command_modules/azure-cli-servicefabric/azure/cli/command_modules/servicefabric/commands.py

```diff
@@ -0,0 +1,49 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
+
+
+from azure.cli.core.commands import cli_command
+from azure.cli.core.util import empty_on_404
+
+from ._client_factory import servicefabric_client_factory
+
+custom_path = 'azure.cli.command_modules.servicefabric.custom#{}'
+mgmt_path = 'azure.mgmt.servicefabric.operations.clusters_operations#{}'
+
+factory = lambda args: servicefabric_client_factory(**args).clusters
+
+cli_command(__name__, 'sf cluster show', mgmt_path.format('ClustersOperations.get'), factory, exception_handler=empty_on_404)
+
+cli_command(__name__, 'sf cluster list', custom_path.format('list_cluster'), factory)
+
+cli_command(__name__, 'sf cluster new', custom_path.format('new_cluster'), factory, no_wait_param=False)
```

Comment:
You don't need `no_wait_param=False`.  If you omit it, it will automatically not enable the `--no-wait` flag.